Tests/ClientTests/GleanPlumbMessageManagerTests.swift

```diff
@@ -10,37 +10,103 @@ import Glean
 class GleanPlumbMessageManagerTests: XCTestCase {
 
     var sut: GleanPlumbMessageManager!
+    var messagingStore: MockGleanPlumbMessageStore!
+    let messageId = "testId"
 
     override func setUp() {
         super.setUp()
-        sut = GleanPlumbMessageManager()
+
         Glean.shared.resetGlean(clearStores: true)
         Glean.shared.enableTestingMode()
+        messagingStore = MockGleanPlumbMessageStore(messageId: messageId)
+        sut = GleanPlumbMessageManager(messagingStore: messagingStore)
     }
 
     override func tearDown() {
         super.tearDown()
+        messagingStore = nil
         sut = nil
     }
 
-    func testOnMessageDisplayed() {
-        if let message = sut.getNextMessage(for: .newTabCard) {
-            sut.onMessageDisplayed(message)
-            testEventMetricRecordingSuccess(metric: GleanMetrics.Messaging.shown)
+    func testManagerOnMessageDisplayed() {
+        guard let message = sut.getNextMessage(for: .newTabCard) else {
```

Comment:
The test are failing on Bitrise but passing locally, I'm investigating why is not getting the message from nimbus.fml